spikingjelly/activation_based/neuron/flexsn.py

```
@@ -12,16 +14,295 @@
+        lowerable_while_loop_scan as _flexsn_lowerable_while_loop_scan,
+    )
+    from ..triton_kernel.flex_sn_inductor import (
+        lowerable_while_loop_scan_final_state as _flexsn_lowerable_while_loop_scan_final_state,
+    )
+    from ..triton_kernel.flex_sn_inductor import (
+        lowerable_scan_available as _flexsn_lowerable_scan_available,
+    )
+    from ..triton_kernel.flex_sn_inductor import (
+        lowerable_while_loop_available as _flexsn_lowerable_while_loop_available,
+    )
+except BaseException as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `BaseException`

(BLE001)